app/assets/images/payment_icons/latitude_buyers_edge_au.svg

```diff
@@ -0,0 +1 @@
AHQ0HG68kOqAcRjRXWAQ8cTKgIH55ckHP1PH2nfZLkXbrdr78SfOzGrCh+9v35fH6HvyzuvAFz8m89shH+uIClaL2hMyWEC8IjTNR0nhLOzFv3t/z64bRJ+rFbLHZdnhO8L56Zzvj6gwL1i210VdcVtcXxsgczeOLrGtzLS41vH/70bbNZ0Kznedrlv+ikAw4EgiCK4/epgqS6k0bXBkgLIhTN7J23u9fn740LyP1gFn8eLMeHMfl6CI3BDMjZj+VFgBFYwrFEKSVns8yDAkN/f1frr9Ws1Oo5s9nzJu3+sXnt+OdTx/VEwTUiohpX3aAqcll0ZBbHEDUJD99/9HCepum0Vo06+4j+tG775cIOchzVgTAA5woJ0jvAIK8SMdOKBKChbGMYWW/RhzAbswF2SUzSk6u/qgqJ8/qHjth9XWUDlQzY9wn7jrWv+nmTpdkBf2WSRmYZ/e7wC8+PGBwQ8QNHpJ0USZFvt2mzLaokO3JyDk+tZTlWf/78afvHDFlZa6C1+GmsaU2L6b2esfA54mfkf/6zq9Msk+kPHBaivH/mN44q++N2l2S22tbpxxaqBP9lpmlY5xSUph97W7w93KyREkAasikT1034ql82P7eJyLKUvnp/lHuqNubjFw8qezy3O1kd6vp3Ue/yWkji3+UqplljhbYCOnBDjzhnfthRv2wSkRL+Yx9LJo6Ofw/b4++1y9tUQl2VMknNrrIiI5xdOjD8YV84LnXD73XvfdDT/B0/M44849kBaa4zhkezBvmcziwV3HxxlF2VCFk88VdsY5snTpcGtIDBu+BG5mPTP1E+Xzzz3Pf9x2pGnAYUoUTJa3w4SuSLuiuKWhYAaYY05+ljs990I0D04ABQ1/30ZisuHPVhWC7LspHZj0fRh/XAUnMeiaVSSshjScmlkGkilEyg34meB4cIV5bVZ9pGdIPXWikfnJrQXvHY9fMwnJbldN2kFY7gMTmFvgeB6qUoDFR5gSllyu3OZJPJqog1Ywojudkc2uVGdCetvQDQ/nMaKSvNOGUZrsvphLzIsL0HPihUKY1mXLnKEpDr1i5L8jlRptB7BNqGacyEAa9hUBCCxpv8c2R6rZvt5Ryw+BNmU5VExuR5ldRaaYVnD9KAyd/UVY4xVnbeAxi8Zr3HjHpwN4+dP8db/E3HhED49bqJODc5L2ge1zqtGq2lauWg4/aK6CvWdF5r7eZWqL5XepxnvOxcCP7zor7dZiuOdQ/kX7++NsjGlOWu7qhuGomlx/JF62Vhm5iIc7MtDPwGH7tuAOj0NM/T3IXgLmvXSY5BmuJCCNfrF/KEvvSq7jSWnlHBRaO9UmMG3th3Ndqki450p2C4OhcW3GjzCM55tgl/ufjHOCVg/n7d73fkbWEfe9kWhvveFNxnLF8rDdqriEeYd1nbdw5xt+jBoe5pj8/QBVzzc7RZ5/CC45KjfrszbyuLWZ8aRe+9ir1WTimvNfQNq3iYFeb008RFO686R+GdNvrg4z1GiTTpmK/Ik20b6FdTO22seTw4rXBeaWzuuKr9qnIee4x0d4FAxS9rwy/v+AOOOK34LeCBOPKPWxfMehv1zinudMPzONbu1bNcrUh92LTGExbNrB5ut1twKGOiTEEXfZZ5q3MCZuUNxgpgEoPl85cv20PHe1r1z5f1qGcM24/M3UC6Q93dQrh8Xt5dDukuxLrZ/M4bcNPaX4Pl+7ZpBeEueMVd5j+lqebp9fyeYbihGpYhUPGn/9uvhodhuDPONubvnfm/zDMKfnruqFZh+yV07uoClk7wPKpxQn6Zx3nW3TKzjleIIwv/9VnHuK0zMRCAfbEfcZFWh9DfiM2DO2vHGi2v/khzB1jIxpsgRqAiH4a7VFKj11mXHRFOwgs/i6fKB5/i/9aSjjyiXQz0F4lvgb6HvMQVaMtreTx+LAZe5XeLnQN8GzPYpysu383QD4LHKdy7c3QX/2M53kpcqPAT/3kEPua8ty2u4R73wYltCWrfYFjJpyH1Y15t3XIYPMbuAMtOnYaIDz73577FiSZd2VJvg1YezSLP1vhuujcYQB6rpc5wJ7nOGsbzOHs/WVwF73gvPof73xZy8AX9aVhfy4TvkWXJG5Qb/UKd8wHirZPADKccCRng4f3sMHHegTH74peIvUK+213kq0nVPyfLy+xo9fMR2v5qzUmn8U1KFl0+kSwMdLB0N5AuPfhoa0uzzH2QSzhz9rb+2rMehgtwHLKDl9JKj7gDJKw+VF548YHd131bQYJihyM9qjMXqszXCrjUa+We8MDDQeLTaXdS+OCXZY0NsuRdfOD6jrQneLTUxk4B+Bi1KifuXQbgBAiAoH/mtm0tbJAMQsmqlSI15AqM5PlF9nDDhrl4Z+kkIl94sxWnkwFMopoqgFfdmLaZM4RDruKpEzCAfdpy4Z/1yZsT6wqCEyacutJPBwdHg78FyaILN1wk4Z86AdwIwsBwrnGLIOKplq/fBrpf8LFiABiZps8Z50jpuIEkQK1t5YzozoLBH+nGh41FutqaczmdwHT7IqTwEVRu9UIscY5btWSd70kg8XrWu1/weqGUU6KPCP/Uq72f1/T4Apjpp5x6AL03PoLS+cYJyAe+4eKn4n2ONFXsJEEC33EaWBk4Kf+KExMvVKm6J02X7PrHQtf86gP02rvCAf5bJ4PvikqGV5a7gQTFC/+a7qA7Kp+LTn74ROQv6gm3Z4RBj8QAAAAASUVORK5CYII="/></g><path d="m35 0h-32a3 3 0 0 0 -3 3v18a3 3 0 0 0 3 3h32a3 3 0 0 0 3-3v-18a3 3 0 0 0 -3-3zm2 20.2a2.8 2.8 0 0 1 -2.8 2.8h-30.4a2.8 2.8 0 0 1 -2.8-2.8v-16.4a2.8 2.8 0 0 1 2.8-2.8h30.4a2.8 2.8 0 0 1 2.8 2.8z" fill="#ab6c3d" fill-rule="evenodd" opacity=".5"/></svg>
```

Comment:
When I zoom in at 200%, the logo looks pixelated. Is there a way to make it vectorized?

[![alt](https://screenshot.click/11-22-8p9gk-ii4ik.png)](https://screenshot.click/11-22-8p9gk-ii4ik.png)

Reply:
Hi @tauthomas01, 
I have excluded this single asset from the pull request for now and would create a separate PR for it. 
Thanks,

Reply:
Please review this PR as our merchants are keen to use other payment icons in the pull request on our gateway.